content/en/blog/2022/enhancements-process-update.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: blog
+title: "Enhancements Opt-in Process Change for v1.26"
+date: 2022-09-06
+slug: enhancements-opt-in
+---
+
+# Enhancements Opt-in Process Change for v1.26 
+
+## Context and Motivations
+Since the inception of the Kubernetes release team, we have used a spreadsheet to keep track of enhancements for the release. The project has scaled massively in the past few years, with almost a hundred enhancements collected for the 1.24 release. This process has become error-prone and time consuming. A lot of manual work is required from the release team and the SIG leads to populate KEPs data in the sheet. We have received continuous feedback from our contributors to streamline the process.
+
+Starting at the beginning of the 1.26 release, we are replacing the sheet with an automated [GitHub project board](https://github.com/orgs/kubernetes/projects/98).
+
+## How does the Github Project Board work?
+
+The board is populated with a script gathering all KEP issues in the `kubernetes/enhancements` repo that have the tag `lead-opted-in`. The enhancements' stage and SIG information will also be automatically pulled from the KEP issue.
+
+
+## What does this mean for the community?
+
+If you are not a SIG lead, nothing will change beside the view of the enhancements collections and the change of platform. KEP authors will continue working with their respective SIG leads to opt in to the release.
```

Comment:
This is okay as is but might be slightly clearer with `... platform change to the Github board.`